lib/matplotlib/image.py

```diff
@@ -9,6 +9,8 @@
 from numbers import Number
 from pathlib import Path
 import urllib.parse
+import urllib.request
```

Comment:
This fails `test_basic.py::test_lazy_imports()`

https://github.com/matplotlib/matplotlib/blob/52761deb15b9c0aca05112befcdefce5cea0a2ad/lib/matplotlib/tests/test_basic.py#L48

#12781 made urllib.request a lazy import.

Reply:
fair enough, I'll put it back.